### DISCUSSION TITLE
PostProcessingSprite Quality

### DISCUSSION THREAD

**Discussion by Abhiram3** [jnsmalm/pixi3d#99] [Q&A]
Is there a way to improve the quality of PostProcessingSprite added to the pixi2d canvas?

Attached the example of normal model load vs PostProcessingSprite .

![sample1](https://user-images.githubusercontent.com/12019238/171613496-d7a4ce4a-76df-4652-bb6f-13ec827f0645.png)
![sample2](https://user-images.githubusercontent.com/12019238/171613505-b7d93fc4-2b5e-41e9-b271-a17d61cdfc87.png)

code for PostProcessingSprite:

`app = new PIXI.Application({
            resizeTo: window,
            backgroundColor: 0xf1f1f1,
            antialias: true,
            // resolution: window.devicePixelRatio
        })

        app.stage.sortableChildren = true;
         const ppsModel = PIXI3D.Model.from(resources[modelName!].gltf);
            ppsObject = new PIXI3D.PostProcessingSprite(app.renderer, {
                width: window.innerWidth,
                height: window.innerHeight,
                objectToRender: pixiModel,
                resolution: 2
            })
            const modelSprite = app.stage.addChild(ppsObject);

            ppsModel.meshes.forEach((mesh: any) => {
                mesh.material.unlit = false; // Set unlit = true to disable all lighting.
                mesh.material.baseColor = new PIXI3D.Color(1, 1, 1, 1); // The base color will be blended together with base color texture (if available).
                mesh.material.alphaMode = "opaque"; // Set alpha mode to "blend" for transparency (base color alpha less than 1).
                mesh.material.exposure = 1.5; // Set exposure to be able to control the brightness.
                mesh.material.metallic = 0; // Set to 1 for a metallic material.
                mesh.material.roughness = 0.3; // Value between 0 and 1 which describes the roughness of the material.
            });`

**Comment by jnsmalm** [comment]
Thanks, I'll investigate this. The problem is due to antialias doesn't work currently on PostProcessingSprite.
